src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -102,6 +102,53 @@ Copyright (c) .NET Foundation. All rights reserved.
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' == ''">%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' != ''">%(LinkBase)\%(Filename)%(Extension)</Link>
+    </AdditionalFiles>
+    
+    <None Update="@(None)">
+      <Link Condition="'%(Link)' == '' And !$([System.IO.Path]::GetFullPath('%(FullPath)').StartsWith('$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)'))\'))">*NeedsLink*</Link>
+
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' != '' And '%(LinkBase)' == ''">%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' != '' And '%(LinkBase)' != ''">%(LinkBase)\%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' == ''">%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' != ''">%(LinkBase)\%(Filename)%(Extension)</Link>
+    </None>
+
+    <Content Update="@(Content)">
+      <Link Condition="'%(Link)' == '' And !$([System.IO.Path]::GetFullPath('%(FullPath)').StartsWith('$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)'))\'))">*NeedsLink*</Link>
+
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' != '' And '%(LinkBase)' == ''">%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' != '' And '%(LinkBase)' != ''">%(LinkBase)\%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' == ''">%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' != ''">%(LinkBase)\%(Filename)%(Extension)</Link>
+    </Content>
+
+    <EmbeddedResource Update="@(EmbeddedResource )">
+      <Link Condition="'%(Link)' == '' And !$([System.IO.Path]::GetFullPath('%(FullPath)').StartsWith('$([System.IO.Path]::GetFullPath('$(MSBuildProjectDirectory)'))\'))">*NeedsLink*</Link>
+
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' != '' And '%(LinkBase)' == ''">%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' != '' And '%(LinkBase)' != ''">%(LinkBase)\%(RecursiveDir)%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' == ''">%(Filename)%(Extension)</Link>
+      <Link Condition="'%(Link)' == '*NeedsLink*' And '%(RecursiveDir)' == '' And '%(LinkBase)' != ''">%(LinkBase)\%(Filename)%(Extension)</Link>
+    </EmbeddedResource >
+  </ItemGroup>
+
```

Comment:
Shouldn't LinkBase work inside the project cone as well? I might want to display/copy in/to a different relative path.